site/content/contribute/getting-started/inactive-contribution.md

```diff
@@ -0,0 +1,26 @@
+---
+title: "Inactive Contributions"
+date: 2017-08-20T12:33:36-04:00
+weight: 1.5
+subsection: Getting Started
+---
+
+# Inactive Contributions
```

Comment:
Because of https://github.com/mattermost/mattermost-developer-documentation/pull/197 this line should be removed.